src/utils/logging-buffer.ts

```diff
@@ -83,35 +89,35 @@ export class LogBuffer {
     if (this.buffer.length === 0) {
       return;
     }
-
+  
     const logsToSend = [...this.buffer];
     this.buffer = [];
-
+  
     try {
-      const response = await fetch(`${this.acrolinxUrl}/int-service/api/v1/logs`, {
-        method: 'POST',
-        headers: {
-          'Content-Type': 'application/json',
-        },
-        body: JSON.stringify({
+      await post<Response>(
```

Comment:
Ideally this should not be part of util logging-buffer. in case if log needs to be on cloud then int-service class should take the responsibility